### PR TITLE
Milestone 3 : validations and model specs

### PR DESCRIPTION
**Pull request by Chrissiku**
## Project requirements 🟢 
- Add the following validations:
  - For the User model:
    - Name must not be blank.
    - PostsCounter must be an integer greater than or equal to zero.
  - For the Post model:
    - Title must not be blank.
    - Title must not exceed 250 characters.
    - CommentsCounter must be an integer greater than or equal to zero.
    - LikesCounter must be an integer greater than or equal to zero.
- Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: spec/models/comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```
Comment (danmainah) [inline spec/models/comment_spec.rb:3]: - [ ] Kindly check on your comment model I got the error below when running the test:
image

File: spec/models/like_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
```
Comment (danmainah) [inline spec/models/like_spec.rb:3]: - [ ] Kindy check on your like model, I encountered the error below when running the tests:
image

File: spec/models/post_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```
Comment (danmainah) [inline spec/models/post_spec.rb:3]: - [ ] Kindy check your post model, I encountered this error when running the tests:
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Chrissiku , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
Hi @Chrissiku 👍🏽, 
Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Suggested changes
_Check the comments under the review._
## There seem to be an error in your migration file, kindly fix it
Screenshot from 2022-09-16 18-48-29

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @Chrissiku 👋, 

Well done for implementing all the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🎯 
✔️ Descriptive PR
✔️ You have written model specs
✔️ No linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chrissiku** [comment]
Thank you @codepantha

**Pull request merged**